scripts/combine_lightcurves.py

```diff
@@ -1,43 +1,93 @@
+#!/usr/bin/env python3
```

Comment:
Per your request for design refactoring advice:  This file is still pretty small, so I'd be reluctant to take the OOAD axe to it until it's doing more work. 

Testing might be cleaner if you wrap it in one class :) 

The only thing I'd consider is separating out the I/O functionality from the light curve combination stuff.